packages/bolt-connection/src/bolt/bolt-protocol-v4x0.js

```diff
@@ -90,7 +90,7 @@ export default class BoltProtocol extends BoltProtocolV3 {
       afterComplete,
       flush = true,
       reactive = false,
-      fetchSize = ALL
+      fetchSize = FETCH_ALL
```

Comment:
Thanks for catching it, I will make sure I will have tests covering this default value.

Reply:
(No action needed here)